cli/parse_pdfs.py

```diff
@@ -70,6 +73,9 @@ def copy_input_to_output_pdf(
 
         try:
             output_path.write_text(blank_output.json(indent=4, ensure_ascii=False))
+        except Exception as e:
+            print(e)
```

Comment:
If we could remove a breakpoint to bring up debugger in production code that would be good. 

Can we also add the error to the _LOGGER statement if we wanna see something